### ISSUE TITLE
Questions about test accuracy

### ISSUE THREAD

**Issue by mt-43** [dk-kim/DFWSGAR#6]
Thanks for your great work! I have run the code and the results show that the basketball dataset is only about 70% accurate, not 75% as in the paper, and I have tried several times without finding out what the problem is.
Can you provide a pre-trained model that can be loaded

**Comment by dk-kim** [comment]
Thank you for your attention to our work. You can use scripts/download_checkpoint.sh to download our pretrained weights.

**Comment by LemonQC** [comment]
> Thanks for your great work! I have run the code and the results show that the basketball dataset is only about 70% accurate, not 75% as in the paper, and I have tried several times without finding out what the problem is. Can you provide a pre-trained model that can be loaded

I found on Volleyball as suffer the same problem

![image](https://user-images.githubusercontent.com/30914380/210127691-545e230f-ab81-465d-bac0-faff0eae6085.png)

**Comment by dk-kim** [comment]
> > Thanks for your great work! I have run the code and the results show that the basketball dataset is only about 70% accurate, not 75% as in the paper, and I have tried several times without finding out what the problem is. Can you provide a pre-trained model that can be loaded
> 
> I found on Volleyball as suffer the same problem
> 
> ![image](https://user-images.githubusercontent.com/30914380/210127691-545e230f-ab81-465d-bac0-faff0eae6085.png)

I think this is the result of our test code, is it the result of our pretrained weight or the result of your own trained weight utilizing our training code? 
You can also refer #1.

**Comment by dk-kim** [comment]
> > Thanks for your great work! I have run the code and the results show that the basketball dataset is only about 70% accurate, not 75% as in the paper, and I have tried several times without finding out what the problem is. Can you provide a pre-trained model that can be loaded
> 
> I found on Volleyball as suffer the same problem
> 
> ![image](https://user-images.githubusercontent.com/30914380/210127691-545e230f-ab81-465d-bac0-faff0eae6085.png)

We find out that we miss some arguments stated in the paper (batch size 4 → 8, weight decay 1e-4 → 1e-3) in the script of training volleyball dataset (train_volleyball.sh). We update the training script and we can obtain the accuracy of 90.5% as stated in the paper. Sorry for confusing you.

**Comment by dk-kim** [comment]
> Thanks for your great work! I have run the code and the results show that the basketball dataset is only about 70% accurate, not 75% as in the paper, and I have tried several times without finding out what the problem is. Can you provide a pre-trained model that can be loaded

We train our model using our training script with two RTX 3090 GPU (24GB) and obtain MCA of 74.16% and 74.48% for two other seeds. If you have any further problems reproducing our work, feel free to ask me.
